docs/how-to/github-settings.rst

```diff
@@ -0,0 +1,60 @@
+.. _github-settings:
+
+GitHub Project Settings
+=======================
+
+Skeleton Based projects
+~~~~~~~~~~~~~~~~~~~~~~~
+
+After making a new project with skeleton (or when forking a skeleton based
+project) the project settings are not copied over.
+
+Project settings are used to provide some of the features and some 
+protection for skeleton based projects. Therefore you should manually apply 
+these settings to any new project:
+
+- Branches -> Add Rule
+  - main
+
+    - tick: Require a pull request before merging
+    - un-tick: Require approvals
+    - tick: Include administrators
+
+  - skeleton
+
+    - tick: Require a pull request before merging
+    - un-tick: Require approvals
+    - tick: Require linear history
+    - tick: Include administrators
```

Comment:
We don't have a skeleton branch in skeleton-bsed projects...

Reply:
right - I only worked this out this morning when updating ibek. removed.